django/db/backends/sqlite3/client.py

```diff
@@ -8,5 +8,5 @@ class DatabaseClient(BaseDatabaseClient):
 
     def runshell(self):
         args = [self.executable_name,
-                self.connection.settings_dict['NAME']]
+                str(self.connection.settings_dict['NAME'])]
```

Comment:
I think this line should include a comment so it is easy to know when it is safe to drop `str()`. See here for an example of how this might look.

https://github.com/django/django/blob/4161e35048d91fa84abf4dd2ebf64e04e0c37ca4/django/db/backends/sqlite3/base.py#L179-L180